src/main/java/org/opentripplanner/model/Trip.java

```diff
@@ -248,8 +249,9 @@ public void setBikesAllowed(BikeAccess bikesAllowed) {
     this.bikesAllowed = bikesAllowed;
   }
 
+  @Override
   public String toString() {
-    return "<Trip " + getId() + ">";
+    return ToStringBuilder.of(this.getClass()).addStr("id", getId().toString()).toString();
   }
 
```

Comment:
The super class implements the toString() in this case - and all (major/aggregate roots) transit entities follow the same pattern: `<[Simple class name] [id]>`. This break with the "newer" toSting style, but there are good reasons.

Since the entire transmodel is "under construction" I would prefer to just revert this change - do not change anything unless you need too.

Reply:
Ok, the original reason why I changed it is that when calling `trip.toString()` in the HTML issue report the `<>` confuses the browser which thinks that is HTML and doesn't show it.

Reply:
Reverted.

Reply:
I can try to enforce these things in the new model.